pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go

```diff
@@ -641,6 +641,86 @@ func TestCSILimits(t *testing.T) {
 	}
 }
 
+func TestCSILimitsAddedPVCQHint(t *testing.T) {
+	tests := []struct {
+		test           string
+		newPod         *v1.Pod
+		attachedVolume *v1.Volume
```

Comment:
Any reason they are separated?
Can we use -
https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/testing/wrappers.go#L414-L418